pom.xml

```diff
@@ -64,6 +64,7 @@
     <module>activiti-spring-connector</module>
     <module>activiti-connector-model</module>
     <module>activiti-spring-application</module>
+    <module>activiti-steps</module>
```

Comment:
@erdemedeiros I think that activiti-core-tests should be ok as the module name.. I like steps as well (but it makes me think of simulation or something not really related with tests).. but I will leave it up to you..

Reply:
Yeah, it's more assertions library than steps library. I'll go with `activiti-core-test`

Reply:
👍